src/cli/command-package.ts

```diff
@@ -29,8 +29,18 @@ export default async function run(options: CliOptions) {
 
   // Ok, we're updating a package in a directory not under our control.
   // We need to be sure it's safe.
-  const {stdout, stderr} = await exec(inDir, 'git', ['status', '-s']);
-  if (!options.force && (stdout || stderr)) {
+  let stdout, stderr;
+  try {
+      ({stdout, stderr} = await exec(inDir, 'git', ['status', '-s']));
+  } catch (e) {
+      ({stdout, stderr} = e);
+  }
+  const isRepo = !((stderr || '').indexOf('Not a git repository') !== -1);
+  if (!isRepo) {
+      console.warn(
+          `Not a git repo, proceeding.`);
+  }
+  if (!options.force && isRepo && (stdout || stderr)) {
```

Comment:
I know we had talked about previously moving options.force up, so it won't even try to do a `git status`

Reply:
That seems good also, but you shouldn't need --force to run in a plain directory